client/blocks/reader-full-post/index.jsx

```diff
@@ -189,7 +189,7 @@ export class FullPostView extends React.Component {
 								siteIcon={ get( site, 'icon.img' ) }
 								feedIcon={ get( feed, 'image' ) }
 								siteName={ post.site_name }
-								siteUrl= { post.site_URL }
+								feedUrl= { get( feed, 'feed_URL' ) }
```

Comment:
This should fall back to the site url, as not all blogs have an associated feed.